src/test/java/athleticli/data/activity/RunTest.java

```diff
@@ -0,0 +1,33 @@
+package athleticli.data.activity;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDateTime;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class RunTest {
+
+    private static final String CAPTION = "Morning Run";
+    private static final int DURATION = 80;
+    private static final int DISTANCE = 18000;
+    private static final LocalDateTime DATE = LocalDateTime.of(2023, 10, 10, 8, 0);
+    private static final int ELEVATION = 60;
+    private Run run;
+
+    @BeforeEach
+    void setUp() {
+        run = new Run(CAPTION, DURATION, DISTANCE, DATE, ELEVATION);
+    }
+
+    @Test
+    void calculateAveragePace() {
+        double averagePace = run.calculateAveragePace();
+        assertEquals(4.444444444444445, averagePace);
```

Comment:
`assertEquals(double expected, double actual)` is deprecated. Use `assertEquals(double expected, double actual, double epsilon)` instead.
```suggestion
        assertEquals(40.0 / 9.0, averagePace, 0.001);
```

Reply:
That's very helpful, thank you!